lib/gui/app/modules/image-writer.js

```diff
@@ -255,6 +255,7 @@ exports.flash = (image, drives) => {
   const analyticsData = {
     image,
     drives,
+    driveAmount: drives.length,
```

Comment:
How about `driveCount` here? `driveAmount` sounds a bit weird to me :D

Reply:
Done